composer.json

```diff
@@ -53,7 +53,8 @@
     "repositories": {
         "markup/rabbitmq-management-api":  {
             "type": "vcs",
-            "url": "https://github.com/usemarkup/php-rabbitmq-management-api"
```

Comment:
Think we could just remove this completely right? its published https://packagist.org/packages/markup/rabbitmq-management-api

Reply:
aw - yeah, sorry - I think i blanked on that - update incoming